README.md

```diff
@@ -282,13 +291,28 @@ Storyblok.get(
     cache: 'no-cache',
     body: JSON.stringify(data),
   }
+  'cdn/stories/home',
+  {
+    version: 'draft',
+  },
+  {
+    mode: 'cors',
+    cache: 'no-cache',
+    body: JSON.stringify(data),
+  }
 )
   .then((response) => {
     console.log(response)
   })
   .catch((error) => {
     console.error(error)
   })
+  .then((response) => {
+    console.log(response)
+  })
+  .catch((error) => {
+    console.error(error)
+  })
```

Comment:
All of this duplicate code got added, probably as part of a botched merged-conflict resolution.